BUILD COMMANDS FOR: lists.psyco

Sun Sep 18 03:38:32 PDT 2005

rm -f lists.pyo lists.pyc
cp /opt/shootout/shootout/bench/lists/lists.psyco lists.py
/usr/bin/python2.4 -OO -c "from py_compile import compile; compile('/opt/shootout/shootout/bench/lists/tmp/lists.py')"
touch lists.psyco_run

=================================================================
COMMAND LINE (%A is single numeric argument):

 /usr/bin/python2.4 lists.pyo %A
Traceback (most recent call last):
  File "/opt/shootout/shootout/bench/lists/tmp/lists.py", line 6, in ?
    import sys, psyco
  File "/usr/lib/python2.4/site-packages/psyco/__init__.py", line 45, in ?
Traceback (most recent call last):
  File "/opt/shootout/shootout/bench/lists/tmp/lists.py", line 6, in ?
    import sys, psyco
  File "/usr/lib/python2.4/site-packages/psyco/__init__.py", line 45, in ?
    raise ImportError, str(e) + extramsg
ImportError: /usr/lib/python2.4/site-packages/psyco/_psyco.so: undefined symbol: PyFPE_jbuf (check that the compiled extension '/usr/lib/python2.4/site-packages/psyco/_psyco.so' is for the correct Python version; this is Python 2.4.1+)
    raise ImportError, str(e) + extramsg
ImportError: /usr/lib/python2.4/site-packages/psyco/_psyco.so: undefined symbol: PyFPE_jbuf (check that the compiled extension '/usr/lib/python2.4/site-packages/psyco/_psyco.so' is for the correct Python version; this is Python 2.4.1+)
TEST FAILED - Child processed exited with error code (1)
TEST FAILED - Child processed exited with error code (1)


PROGRAM OUTPUT
==============
